src/workerd/api/actor.h

```diff
@@ -179,6 +188,13 @@ class DurableObjectNamespace: public jsg::Object {
       JSG_METHOD(getExisting);
     }
     JSG_METHOD(jurisdiction);
+    if (flags.getDurableObjectDestroy()){
+      if (flags.getDurableObjectGetExisting()) {
```

Comment:
IIRC we're adding this method specifically for one internal customer, the same one that is also using `getExisting`. Is it important to support this in the case that `getExisting` isn't enabled?

IMO the meaning of `delete` without `getExisting` gets awkward, since the next `get` will recreate the object.

(I also still feel that we shouldn't introduce either `delete` or `getExisting` more broadly -- this is a hack for one user who is already far off the beaten path with their use of DO.)